usbsdmux/usbsdmux.py

```diff
@@ -92,6 +92,7 @@ def __init__(self, sg):
         sg -- /dev/sg* to use
         """
         self._pca = Pca9536(sg)
+        self.type = UsbSdMuxFactory.SD_MUX_TYPE_LEGACY
```

Comment:
The class is already a unique object, so there's no need to for this indirection.